uber/templates/registration/register.html

```diff
@@ -9,10 +9,11 @@
     window.setTimeout(toastr.clear, 10000);
 
     var maybeBold = function() {
-        if ($.val('payment_method') === {{ c.MANUAL }}) {
-            $('#email').css('font-weight', 'bold');
+        var $emailLabel = $('#email').parents('.form-group').find('label');
+        if (!$.field('payment_method') || $.val('payment_method') === {{ c.MANUAL }}) {
+            $emailLabel.addClass('optional-field');
         } else {
-            $('#email').css('font-weight', 'normal');
+            $emailLabel.removeClass('optional-field');
```

Comment:
Since we can no longer rely on the "Payment Method" field always being present on this page, I added an if check to avoid Javascript errors.